docs/README.md

```diff
@@ -65,6 +65,13 @@ Note that the `slider-layout` block, exported from the Slider Layout app, is giv
 | `images`  | `array`  | Array of objects declaring all desired images to be rendered. | `undefined`   |
 | `height`  | `number` | Image height for all images declared in the `image` object.   | `undefined`   |
 
+### `image-list` props
+
+| Prop name | Type     | Description                                                   | Default value |
+| --------- | -------- | ------------------------------------------------------------- | ------------- |
+| `images`  | `array`  | Array of objects declaring all desired images to be rendered. | `undefined`   |
+| `height`  | `number` | Image height for all images declared in the `image` object.   | `undefined`   |
```

Comment:
Is it in `px`?

Reply:
It's the same interface as the `list-context.image-list`